### DISCUSSION TITLE
[M-Z] (2024/25) - sezione 0.10 es 4.19 (13 novembre)

### DISCUSSION THREAD

**Discussion by alem1105** [sapienzastudentsnetwork/calcolo-delle-probabilita#322] [Esercizi M-Z]
![immagine](https://github.com/user-attachments/assets/14447e18-8d57-485f-91df-71a26610bec3)

**Comment by rimaout** [comment]
Calcolare la densità di probabilità discreta $P_{X}(a)$

$$
P_{X}(a) = \begin{cases}
\frac{1}{2} & \text{se }a = 0\\
\frac{3}{5}- \frac{1}{2} & \text{se }a = 1 \\
\frac{4}{5}- \frac{3}{5} & \text{se }a = 2 \\
\frac{9}{10}- \frac{4}{5} & \text{se }a = 3 \\
\ 1 - \frac{9}{10} & \text{se }a = 3.5
\end{cases}
$$

**oss:**  La somma delle probabilità esce $\left( \frac{1}{2} \right) + \left( \frac{3}{5}- \frac{1}{2} \right) + \left( \frac{4}{5}- \frac{3}{5} \right) + \left(\frac{9}{10}- \frac{4}{5}\right) + \left( 1 - \frac{9}{10} \right) = 1$ quindi dovrebbe essere giusto. credo / spero :)

**Comment by marcoseverini** [comment]
![immagine](https://github.com/user-attachments/assets/ea2fb194-feac-45b8-8e17-b67f01a5e861)
![immagine](https://github.com/user-attachments/assets/aa4d5c45-6ae6-41d2-8b67-f4e988f0e1dd)
